README.md

```diff
@@ -1,176 +1,79 @@
-
-#### Swift
-`let success = VIMDeeplink.openVimeoApp()`
-
-#### Objective C
-`BOOL success = [VIMDeeplink openVimeoApp];`
-
-## View a video in the Vimeo app
-
-You must provide a video uri to this method. You can find a video's uri by making an API call to our [video endpoints](https://developer.vimeo.com/api/endpoints/videos#/{video_id}).
-
-#### Swift
-`let success = VIMDeeplink.showVideo(videoURI: "...")`
-
-#### Objective C
-`BOOL success = [VIMDeeplink showVideoWithURI:@"..."]; // URI obtained via the Vimeo API`
-
-## View a user in the Vimeo app
-
-You must provide a user uri to this method. You can find a user's uri by making an API call to our [user endpoints](https://developer.vimeo.com/api/endpoints/users#/{user_id}).
-
-#### Swift
-`let success = VIMDeeplink.showUser(userURI: "...")`
-
-#### Objective C
-`BOOL success = [VIMDeeplink showUserWithURI:@"..."];`
-
-## View a category in the Vimeo app
-
-You must provide a category uri to this method. You can find a category's uri by making an API call to our [category endpoints](https://developer.vimeo.com/api/endpoints/categories).
-
-#### Swift
-`let success = VIMDeeplink.showCategory(categoryURI: "...")`
-
-#### Objective C
-`BOOL success = [VIMDeeplink showCategoryWithURI:@"..."];`
-
-## View a channel in the Vimeo app
-
-You must provide a channel uri to this method. You can find a channel's uri by making an API call to our [channel endpoints](https://developer.vimeo.com/api/endpoints/channels).
-
-#### Swift
-`let success = VIMDeeplink.showChannel(userURI: "...")`
-
-#### Objective C
-`BOOL success = [VIMDeeplink showChannelWithURI:@"..."];`
-
-## Open the all categories page
-
-#### Swift
-`let success = VIMDeeplink.showAllCategories()`
-
-#### Objective C
-`BOOL success = [VIMDeeplink showAllCategories];`
-
-## Open the Explore tab
-
-#### Swift
-`let success = VIMDeeplink.showExplore()`
-
-#### Objective C
-`BOOL success = [VIMDeeplink showExplore];`
+### View Vimeo iOS in the App Store
 
-## Open the Feed tab
+```Swift
+let success = VimeoDeeplinkOpener.viewVimeoAppInAppStore()
+```
 
-#### Swift
-`let success = VIMDeeplink.showFeed()`
+### Check if the Vimeo iOS app is installed
 
-#### Objective C
-`BOOL success = [VIMDeeplink showFeed];`
+```Swift
+let isInstalled = VimeoDeeplinkOpener.isVimeoAppInstalled()
+```
 
-## Open the Playlists tab
+### Open the Vimeo iOS app
 
-#### Swift
-`let success = VIMDeeplink.showPlaylists()`
+```Swift
+let success = VimeoDeeplinkOpener.openVimeo()
+```
 
-#### Objective C
-`BOOL success = [VIMDeeplink showPlaylists];`
+### Open all categories
 
-## Open the Upload from camera roll view
+```Swift
+let success = VimeoDeeplinkOpener.openCategories()
+```
 
-#### Swift
-`let success = VIMDeeplink.showUpload()`
+### Open the video player
 
-#### Objective C
-`BOOL success = [VIMDeeplink showUpload];`
+```Swift
+let uri = "/videos/2" // Where `uri` is obtained from the Vimeo API
+let success = VimeoDeeplinkOpener.openVideoPlayer(uri: uri)
```

Comment:
I like that you have a ```uri``` but it may be more meaningful to have a Vimeo video represented here. I use ```149058362``` in the Android library example, which is a Vimeo Jobs video.